scripts/db-invariant-sweeps/predicates/section9-po-ap-controls.sql

```
@@ -48,14 +75,21 @@ UNION ALL
-   OR has_table_privilege('authenticated', 'public.vendors', 'DELETE')
-   OR has_table_privilege('authenticated', 'public.vendors', 'TRUNCATE')
-   OR has_table_privilege('anon', 'public.vendors', 'INSERT')
-   OR has_table_privilege('anon', 'public.vendors', 'UPDATE')
-   OR has_table_privilege('anon', 'public.vendors', 'DELETE')
-   OR has_table_privilege('anon', 'public.vendors', 'TRUNCATE')
+-- has_table_privilege() with a role NAME raises `role "<name>" does not exist`
+-- when that role is absent. The error aborts the whole statement, so every other
+-- arm of this UNION ALL returns nothing and the sweep reports a false CLEAN.
+-- Test pg_roles first — the same shape migration 20260831235900 uses for
+-- has_function_privilege.
+WHERE (EXISTS (SELECT 1 FROM pg_roles WHERE rolname = 'authenticated')
```

SQLFluff (4.3.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)